spec/models/services/facebook_spec.rb

```diff
@@ -22,39 +22,49 @@
       post_params = @service.create_post_params(post)
     end
 
-    it 'does not add post link when no photos' do
+    it "adds (via <post URL>) when message has no URLs " do
```

Comment:
> I think all comments are addressed now...

The `when message has no URLs ` can be removed here, but otherwise it's OK :+1:

Reply:
Is there a way to do that through here?  This death by a thousand cuts thing is killing me.